guides/resource-provisioning/servicebus-queues.md

```diff
@@ -0,0 +1,138 @@
+# Azure Service Bus Queues
+
+This guide describes how to add Azure Service Bus queues to a microservice running on the FFC Platform Kubernetes cluster.
```

Comment:
Similar to Postgres don't think we should say Service Bus runs in Kubernetes.  We could just say Azure Service Bus.

Reply:
Updated language